core/shared/src/main/scala/scalaz/zio/Ref.scala

```diff
@@ -16,7 +16,7 @@ import java.util.concurrent.atomic.AtomicReference
  * } yield ()
  * }}}
  */
-final class Ref[A] private (private val value: AtomicReference[A]) extends AnyVal {
+final class Ref[A] private (private val value: AtomicReference[A]) extends AnyVal with Serializable {
```

Comment:
Does this work, even though it still extends `AnyVal`?

Reply:
Yes oddly it works! if I do not extend from ```Serializable``` then my unit test fails though. It may be because I think primitives are by default serializable, but we still need to extend here for it to work